apps/tilbakemelding-api/README.md

```diff
@@ -0,0 +1,10 @@
+# testnorge-tilbakemelding-api
+App for innsendinger tilbakemeldinger. Tilbakemeldingene blir publisert i slack
```

Comment:
App for innsending av tilbakemeldinger. Tilbakemeldingene blir publisert i slack